README.md

```diff
@@ -4,6 +4,26 @@ This project is part of ASP.NET Core. You can find samples, documentation and ge
 
 Use https://github.com/aspnet/AspNetCore/issues for issues with this project.
 
+## Build this library
+
+There are multiple ways to build this library
```

Comment:
What do you think about calling this section "Use this library" instead of "Build this library"? I know you're technically building it with vcpkg.

Reply:
I don't think so, this is all about building. Using is below at "Example usage"

Reply:
How about "Install this library"? Maybe it's different for C++, but I think most people who just want to use a library won't initially look in "Build this library" section unless they want to modify it.